code/game/machinery/autolathe/autolathe.dm

```diff
@@ -74,6 +74,7 @@
 	var/use_oddities = FALSE
 	var/datum/component/inspiration/inspiration
 	var/obj/item/oddity
+	var/use_license = TRUE
 	var/is_nanoforge = FALSE
```

Comment:
`is_nanoforge` variable is no longer in use. May be it should be removed?

Reply:
this variable is still used, it is just relegated to .tmpl files.